### DISCUSSION TITLE
Feature parity between `Window::` and `MenuBar::` (but especially `resizeable()`)

### DISCUSSION THREAD

**Discussion by nexxai** [NativePHP/laravel#159] [Ideas]
It would be really great if `MenuBar::` apps could have their window sizes locked using `resizeable(false)` like its `Window::` brother can.

**Comment by mpociot** [comment]
Is the menubar window resizable for you? I think this should probably always be false?
Which OS are you on?

**Reply by nexxai** [reply]
I'm on macOS; the MenuBar app is always resizeable for me.  

EDIT: And I'm just re-reading my original post and not sure if I was clear about what I'm asking for: I really want MenuBar apps to have the option of *not* being resizeable.  They currently *are* resizeable and I do not want that.

**Reply by MasterRO94** [reply]
Yep, same for me. MacOS Intel. MenuBar is resizable w/o the ability to disable this behavior.

**Comment by simonhamp** [comment]
This was just resolved in [`nativephp/laravel v0.6.3`](https://github.com/NativePHP/laravel/releases/tag/0.6.3) and [`nativephp/electron v0.8.6`](https://github.com/NativePHP/electron/releases/tag/0.8.6).
